Functions/Initialization/xASL_init_InitializeMutex.m

```diff
@@ -1,4 +1,4 @@
-function [x] = xASL_init_InitializeMutex(x, ModuleName)
+function [x, bLocked] = xASL_init_InitializeMutex(x, ModuleName)
 %xASL_init_InitializeMutex Initialize mutex system for ExploreASL modules
 %
 % FORMAT: [x] = xASL_init_InitializeMutex(x, ModuleName)

```

Comment:
Please modify format and output

Reply:
See my other comment and revert this added bLocked parameter.

Reply:
Now throughout bLocked -> x.mutex.bAnyModuleLocked

Reply:
Sounds good. Will check and test.